### ISSUE TITLE
Add way to look at automation per WCAG SC

### ISSUE THREAD

**Issue by MelSumner** [MelSumner/a11y-automation#505]
I think we could add a feature that groups data in a different way. Carrie Fisher once asked if there was a way to show the different automation options per WCAG WC. That would be a useful feature, I think. Implementation would need to be thoughtful enough so it didn't imply coverage %

If it does suggest coverage amount in some way, it should be accurate.
